lib/cell.rb

```diff
@@ -0,0 +1,51 @@
+require './lib/ship'
+
+class Cell
+  attr_reader :coordinate, :ship
+
+  def initialize(coordinate)
+    @coordinate     = coordinate
```

Comment:
I'm glad you accepted the format regarding aligned equal signs. I have received many compliments for those aesthetics.